tests/fsharpqa/Source/nunitConf.fs

```diff
@@ -0,0 +1,420 @@
+module NUnitConf
+
+open System
+open System.IO
+open NUnit.Framework
+
+open UpdateCmd
+open TestConfig
+open PlatformHelpers
+open FSharpTestSuiteTypes
+
+let checkTestResult result =
+    match result with
+    | Success () -> ()
+    | Failure (GenericError msg) -> Assert.Fail (msg)
+    | Failure (ProcessExecError (err, msg)) -> Assert.Fail (sprintf "ERRORLEVEL %i %s" err msg)
+    | Failure (Skipped msg) -> Assert.Ignore(sprintf "skipped. Reason: %s" msg)
+
+let checkResult result = 
+    match result with
+    | CmdResult.ErrorLevel err -> let x = err, (sprintf "ERRORLEVEL %d" err) in Failure (RunError.ProcessExecError x)
+    | CmdResult.Success -> Success ()
+
+let skip msg () = Failure (Skipped msg)
+let genericError msg () = Failure (GenericError msg)
+let errorLevel exitCode msg () = Failure (ProcessExecError (exitCode,msg))
+
+let envVars () = 
+    System.Environment.GetEnvironmentVariables () 
+    |> Seq.cast<System.Collections.DictionaryEntry>
+    |> Seq.map (fun d -> d.Key :?> string, d.Value :?> string)
+    |> Map.ofSeq
+
+let defaultConfigurationName =
+#if !DEBUG
+    DEBUG
+#else
+    RELEASE
+#endif
+
+let parseConfigurationName (name: string) =
+    match name.ToUpper() with
+    | "RELEASE" -> RELEASE
+    | "DEBUG" -> DEBUG
+    | s -> failwithf "invalid env var FSHARP_TEST_SUITE_CONFIGURATION '%s'" s
+    
+
+let initializeSuite () =
+
+    let configurationName = defaultConfigurationName
+
+    let doNgen = true;
+
+    let FSCBinPath = __SOURCE_DIRECTORY__/".."/".."/(sprintf "%O" configurationName)/"net40"/"bin"
+
+    let mapWithDefaults defaults m =
+        Seq.concat [ (Map.toSeq defaults) ; (Map.toSeq m) ] |> Map.ofSeq
+
+    let env = 
+        envVars ()
+        |> mapWithDefaults ( [ "FSCBINPATH", FSCBinPath ] |> Map.ofList )
+
+    let configurationName =
+        match env |> Map.tryFind "FSHARP_TEST_SUITE_CONFIGURATION" |> Option.map parseConfigurationName with
+        | Some confName -> confName
+        | None -> configurationName
+
+    processor {
+//        do! updateCmd env { Configuration = configurationName; Ngen = doNgen; }
+//            |> Attempt.Run
+//            |> function Success () -> Success () | Failure msg -> genericError msg ()
+
+        let cfg =
+            let c = config env
+            let usedEnvVars =
+                c.EnvironmentVariables 
+                |> Map.add "FSC" c.FSC             
+            { c with EnvironmentVariables = usedEnvVars }
+
+        logConfig cfg
+
+//        let directoryExists = Commands.directoryExists (Path.GetTempPath()) >> Option.isSome 
```

Comment:
i'll add doc, it's the same as fsharp suite, 
- run updatecmd (the f# version of update.cmd )
- read config (what fsc/fsi, path for peverify/etc)
 - some smoke tests (run fsc/fsi to check works, otherwise it's useless run the suite)